OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -121,24 +130,33 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
             iconDrawable = new IconDrawable(context, FontAwesomeIcons.fa_comment)
                     .sizeRes(context, R.dimen.edx_small)
                     .colorRes(context, R.color.edx_brand_gray_base);
+            holder.discussionCommentCountReportTextView.setVisibility(View.VISIBLE);
```

Comment:
This discussion has taken place in the past too, but since the layout of both isn't complex and is almost the same currently, we've always chosen to avoid the refactor for now.